.vscode/tasks.json

```diff
@@ -38,6 +37,18 @@
             "problemMatcher": [
                 "$tsc"
             ]
+        },
+        {
+            "label": "test",
```

Comment:
Test task with the "test" group kind. This allows us to assign a keyboard shortcut to the task rather than running it manually with the command pallet or even triggering from a terminal.